components/automate-cli/cmd/chef-automate/iam.go

```diff
@@ -93,17 +95,20 @@ func newIAMUpgradeToV2Cmd() *cobra.Command {
 		RunE: runIAMUpgradeToV2Cmd,
 		Args: cobra.ExactArgs(0),
 	}
+	cmd.PersistentFlags().BoolVar(
+		&iamCmdFlags.skipLegacyUpgrade,
+		"skip-policy-migration",
+		false,
+		"Do not migrate policies from IAM v1.")
 	cmd.PersistentFlags().BoolVar(
 		&iamCmdFlags.betaVersion,
 		"beta2.1",
 		false,
 		"Upgrade to version 2.1 with beta project authorization.")
-	err := cmd.PersistentFlags().MarkHidden("beta2.1")
-	// we could also ignore the lint error :shrug:
-	if err != nil {
-		fmt.Printf("failed configuring cobra: %s\n", err.Error())
-		panic(err.Error())
-	}
+
+	// all flags are hidden right now
+	cmd.PersistentFlags().VisitAll(func(f *pflag.Flag) { f.Hidden = !isDevMode() })
```

Comment:
I thought the team consensus was that `skip-policy-migration` was for public consumption as well...?

Reply:
i believe that consensus convo happened last friday during subtasking and i wasnt there. if we do make it public we should have a corresponding docs update, so id like to do that in a separate pr if possible.

Reply:
I'm all in favour of not adding anything here 😉